examples/name-service/programs/name-service/tests/test.rs

```diff
@@ -3,15 +3,16 @@
 use std::net::{Ipv4Addr, Ipv6Addr};
 
 use anchor_lang::{AnchorDeserialize, InstructionData, ToAccountMetas};
+use forester_utils::indexer::Indexer;
+use forester_utils::rpc::RpcConnection;
```

Comment:
Not critical but we could also reexport `forester_utils` in `test_utils`.

Reply:
[Done](https://github.com/Lightprotocol/light-protocol/pull/1133/commits/ba9ccea78807890cbc59be95844b49e8143414f7#diff-19ce4320e3cd1ac4e16e3b5a99eb75686772499bde4234d7379d65dbe63a0549R22-R36)